src/processors/typescript.ts

```diff
@@ -10,7 +16,13 @@ export default (options: Options.Typescript): PreprocessorGroup => ({
     );
     if (lang !== 'typescript') return { code: content };
 
-    const transformed = await transformer({ content, filename, options });
+    const transformed = await transformer({
+      content,
+      filename,
+      markup: markupCache[svelteFile.filename],
```

Comment:
Hi, as a suggestion, I'd move the additional parameter to the end of the param list to be consistent with your additions to other param lists earlier in the file.

Reply:
Done.